content/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -178,177 +80,24 @@ of the altered protocol.
+no external dependencies for the protocol implementation. Nonetheless, the GossipSub protocol supports: i) Peer Exchange, which allows applications to bootstrap from a known set of bootstrap peers without an external peer discovery mechanism, and ii) Explicit Peering Agreements, where the application can specify a list of peers to which nodes should connect when joining.
 
-- While in this state:
-  - `ChainSync` is well-bootstrapped, and has an initial **trusted** `StateTree` to start from.
-  - `ChainSync` fetches and validates blocks.
-  - `ChainSync` is receiving and validating latest `Blocks` from `BlockPubsub`
-  - `ChainSync` DOES NOT have unvalidated blocks between `ChainSync.FinalityTipset` and `ChainSync.TargetHeads`
-  - `ChainSync` MUST drop back to another state if security conditions change.
-  - Keep a set of gap measures:
-    - `BlockGap` is the number of remaining blocks to validate between the Validated blocks and `BestTargetHead`.
-      - (ie how many epochs do we need to validate to have validated `BestTargetHead`, does not include null blocks)
-    - `EpochGap` is the number of epochs between the latest validated block, and `BestTargetHead` (includes null blocks).
-    - `MaxBlockGap = 2`, which means how many blocks may `ChainSync` fall behind on before switching back to `CHAIN_CATCHUP` (does not include null blocks)
-    - `MaxEpochGap = 10`, which means how many epochs may `ChainSync` fall behind on before switching back to `CHAIN_CATCHUP` (includes null blocks)
-- **Chain State and Finality**:
-  - In this state, the **chain MUST advance** as all the blocks up to `BestTargetHead` are validated.
-  - New blocks are finalized as they cross the finality threshold (`ValidG.Heads[0].ChainEpoch - FinalityLookback`)
-  - New finalized blocks are reported to consumers.
-  - The chain state provided includes the `Blocks` and `StateTree` for the `Finality` epoch, as well as
-    candidate `Blocks` and `StateTrees` for unfinalized epochs.
-- **security conditions to transition out:**
-  - Temporary network partitions (see _Detecting Network Partitions_).
-  - Encounter gaps of `>MaxBlockGap` or `>MaxEpochGap` between Validated set and a new `ChainSync.BestTargetHead`
-- **transitions out:**
-  - if a temporary network partition is detected: move to `CHAIN_CATCHUP`
-  - if `BlockGap > MaxBlockGap`: move to `CHAIN_CATCHUP`
-  - if `EpochGap > MaxEpochGap`: move to `CHAIN_CATCHUP`
-  - if node is shut down: move to `INIT`
+### Peer Exchange
 
-## Block Fetching, Validation, and Propagation
+This process is supported through either bootstrap nodes or other normal peers. Bootstrap nodes must be maintained by system operators. They have to be stable and operate independently of protocol constructions, such as the GossipSub mesh construction, that is, bootstrap nodes do not maintain connections to the mesh.
```

Comment:
I think this basically replaces the previous paragraph I commented on.

Reply:
I would mention bootstrap nodes in a single paragraph to make sure it has enough emphasis (fine if the end result is two paragraphs after that refactor).